### ISSUE TITLE
Events not present in pmaps collection (PCity)

### ISSUE THREAD

**Issue by jerenner** [next-exp/IC#436]
Going back and re-running the analysis for the old 7-bar Th/Cs runs with the latest master branch (after the refactoring for the new PMap format), a "KeyError" is occurring in the event_loop function of class PCity, at the line:
```
pmap = pmaps[evt_number]
```
Apparently when it gets the event numbers, timestamps, and PMaps to be processed from the pmapVectors, some of the events in the list of event numbers are not actually in the collection of PMaps.  Perhaps this is an error in how the old files were saved, but as a temporary fix I've placed just above this line:
```
if(not (evt_number in pmaps)): continue
```
Is this something we should have in the code, or does this seem like a problem with how these particular files were saved?

**Comment by gonzaponte** [comment]
This is addressed in #432. Please someone merge.

**Comment by gonzaponte** [comment]
Closed via #432.
